src/Hedgehog/Gen.fs

```diff
@@ -368,13 +355,13 @@ module Gen =
 
     // Generates a random character in the specified range.
     let charRange (lo : char) (hi : char) : Gen<char> =
-        range (int lo) (int hi) |> map char
+        integral <| Range.constant (int lo) (int hi) |> map char
 
     /// Generates a random character.
     let char : Gen<char> =
         let lo = System.Char.MinValue
         let hi = System.Char.MaxValue
-        sizedRange (int lo) (int hi) |> map char
```

Comment:
`char` doesn't seem to work with `Range`

![image](https://cloud.githubusercontent.com/assets/287532/25435264/1e6b302c-2a98-11e7-8fe8-e344a93a5659.png)

So I've changed our existing ones to match the ones we have in Haskell. Done in 41cde3c.